content/en/tracing/profiler/enabling/native.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Enabling the Native Profiler for Linux
+kind: Documentation
+code_lang: native
+type: multi-code-lang
+code_lang_weight: 40
+further_reading:
+    - link: 'getting_started/profiler'
+      tag: 'Documentation'
+      text: 'Getting Started with Profiler'
+    - link: 'tracing/profiler/profiler_troubleshooting'
+      tag: 'Documentation'
+      text: 'Fix problems you encounter while using the profiler'
+---
+
+{{< site-region region="us5" >}}
+<div class="alert alert-warning">
+  The Continuous Profiler is not available for the Datadog {{< region-param key="dd_site_name" >}} site.
+</div>
+{{< /site-region >}}
+
+## Requirements
+
+The Datadog Profiler requires Linux kernel v4.17+ on an `amd64` compatible processor. It does not currently support macOS, BSD, Windows, or other operating systems besides Linux v4.17 or later. `/proc/sys/kernel/perf_event_open` must be set to *at most* 2 or `CAP_SYS_ADMIN` (or `CAP_PERFMON` on Linux v5.8 or later) must be granted to the profiling process or service container.
+
+## Installation
+
+To begin profiling applications:
+
+1. Ensure you are running the Datadog agent version [7.20.2][1]+ or [6.20.2][2]+.
+
+2. Download the appropriate [ddprof binary][3] for your Linux distribution.
+
+3. Modify your service invocation to include the profiler. Your usual command is passed as the last arguments to the `ddprof` executable.
+   {{< tabs >}}
+{{% tab "Environment variables" %}}
+
+```shell
+export DD_ENV="<APPLICATION_ENVIRONMENT>"
+export DD_SERVICE="<SERVICE_NAME>"
+export DD_VERSION="<APPLICATION_VERSION>"
+ddprof myapp --arg1 --arg2
+```
+
+The following parameters are recommended, as they help identify the source of your profiles in the Profiling UI:
+- `DD_ENV`, the [environment][4] name, for example, `production`.
+- `DD_SERVICE`, the [service][4] name, for example, `web-backend`. If this is not specified, the profiler will use the default value `myservice`.
+- `DD_VERSION`, the [version][4] of your application.
```

Comment:
2 cents, just put an obvious example in the snippet above and leave the full explanation for the config table below. Eg https://github.com/DataDog/documentation/blob/master/content/en/tracing/profiler/enabling/ruby.md?plain=1#L52-L55

Reply:
Good idea.  Done.

Reply:
I'm collapsing it a bit further.